Objects/Converters/ConverterRevit/ConverterRevitShared/Extensions/ForgeTypeIdExtensions.cs

```diff
@@ -24,7 +25,24 @@ public static class ForgeTypeIdExtensions
     }
     public static string ToUniqueString(this ForgeTypeId forgeTypeId)
     {
-      return forgeTypeId.TypeId.ToString();
+      return forgeTypeId.TypeId;
+    }
+    private readonly static HashSet<string> lengthTypes = new()
+    {
+      UnitTypeId.Millimeters.ToUniqueString(),
+      UnitTypeId.Centimeters.ToUniqueString(),
+      UnitTypeId.Meters.ToUniqueString(),
+      UnitTypeId.Inches.ToUniqueString(),
+      UnitTypeId.Feet.ToUniqueString(),
+      UnitTypeId.FeetFractionalInches.ToUniqueString(),
+    };
+    public static bool IsLengthType(this ForgeTypeId forgeTypeId)
```

Comment:
If this is the conditional flag to switch on I'd prefer to flag IsLinearConversion.

There are lots of non Length units that will be linear conversions.